.github/workflows/e2e-test.yml

```diff
@@ -70,7 +82,8 @@ jobs:
       - name: Create opt/ab2d directory and download keystore
         run: |
           mkdir -p opt/ab2d
-          aws s3 cp s3://ab2d-east-impl-main/ab2d_imp_keystore $AB2D_BFD_KEYSTORE_LOCATION
+          KEYSTORE_FILE_NAME="ab2d_${{ inputs.environment == 'test' && 'imp' || inputs.environment }}_keystore"
```

Comment:
Shoudlnt we be consistent with "Impl" usage.. in som places "Imp" and some other place "Impl" is a lil confusing. Minor change if thats okay with you

Reply:
Also  what do you think about a check for valid/passed E2E pipeline is a pre-requisite to deploy?

Reply:
Yes we could improve consistency in the naming of that keystore in a future ticket.

We need to deploy to impl/test before running the e2e test, so it can't be a pre-requisite. We should have a passing e2e test before running the release pipeline. We could discuss ways to require that.